consensus/istanbul/validator/weighted.go

```diff
@@ -629,19 +677,15 @@ func (valSet *weightedCouncil) Refresh(hash common.Hash, blockNum uint64, stakin
 				weight = 1
 			}
 			atomic.StoreInt64(&weightedVal.weight, weight)
+			localLogger = localLogger.NewWith(weightedVal.String(), weight)
```

Comment:
It assigns `localLogger` in the loop, is this intended?

Reply:
yes it is. It is to make a log of weight as one sentence